internal/pkg/cli/job_init.go

```diff
@@ -143,9 +151,101 @@ func (o *initJobOpts) Ask() error {
 
 // Execute writes the job's manifest file and stores the name in SSM.
 func (o *initJobOpts) Execute() error {
+	app, err := o.store.GetApplication(o.appName)
+	if err != nil {
+		return fmt.Errorf("get application %s: %w", o.appName, err)
+	}
+
+	manifestPath, err := o.createManifest()
+	if err != nil {
+		return err
+	}
+	o.manifestPath = manifestPath
+
+	o.prog.Start(fmt.Sprintf(fmtAddJobToAppStart, o.name))
+	if err := o.appDeployer.AddJobToApp(app, o.name); err != nil {
+		o.prog.Stop(log.Serrorf(fmtAddJobToAppFailed, o.name))
+		return fmt.Errorf("add job %s to application %s: %w", o.name, o.appName, err)
+	}
+	o.prog.Stop(log.Ssuccessf(fmtAddJobToAppComplete, o.name))
```

Comment:
Hmm. I'm not sure; the messages are specifically about the ECR repo, so more deployer than store? `app_init` and `svc_init` both have the ordering like this, for what it's worth.

Reply:
I think this line staying here is alright since it is to indicate we successfully add the job to the app which under the hood just update the app stack set (should finish after [this](https://github.com/aws/copilot-cli/pull/1423/files/adb391a2a9761efdfd856839407635041b39dceb#diff-d5f7ba23a93d85624efe93707ecfd5e0R166)).